main.py

```
@@ -231,6 +244,72 @@ async def cli_loop(sk, pk, chain, mempool, network):
             else:
                 print("  ❌ Transaction rejected (invalid sig, duplicate, or mempool full).")
 
+        # ── deploy ──
+        elif cmd == "deploy":
+            if len(parts) < 2:
+                print("  Usage: deploy <filepath> [amount] [fee]")
+                continue
+            filepath = parts[1]
+            try:
+                with open(filepath, "r") as f:
```

Ruff (0.15.15)

[warning] Async functions should not open files with blocking methods like `open`

(ASYNC230)